package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "forta-agent-starter",
+  "version": "0.0.1",
+  "description": "Forta Agent Typescript starter project",
+  "repository": "https://github.com/forta-network/forta-bot-sdk/tree/master/starter-project",
+  "chainIds": [
+    1
+  ],
```

Comment:
this should include chainIds that your bot is supposed to work on. so if you want to deploy your bot on polygon and ethereum for example, this should include 1 and 137. otherwise it wouldn't work in production.